### ISSUE TITLE
Will you release a pytorch version of the code to train the encoder?

### ISSUE THREAD

**Issue by doutzi** [genforce/idinvert_pytorch#11]
Hi, I was wondering if you plan to release a PyTorch version of the code necessary to train the encoder? That would be great.

**Comment by ShenYujun** [comment]
Yes. Please keep an eye on the [genforce](https://github.com/genforce/genforce) repo.

**Comment by Hsintien-Ng** [comment]
I have reproduced the train_encoder.py myself according to the tf version. I found an interesting thing that the learning rate in pytorch version should be smaller than 1e-4 while the one in tf version could be 1e-4. Also, the results of training with lr=1e-5 seems not as good as that in tf version.

**Comment by doutzi** [comment]
Hi @Hsintien-Ng, could you possibly share your reproduced train_encoder.py? That would be very nice :) I'm not familiar with tf and therefore I don't feel confident to reproduce it myself, but I'd very much like to experiment with the encoder training.

**Comment by Hsintien-Ng** [comment]
Yep, @doutzi, please follow the reproduced project in https://github.com/Hsintien-Ng/idinvert_pytorch-reproduced. If there exists some errors, please point out them.
